plugins/hypervisors/xenserver/src/com/cloud/hypervisor/xenserver/resource/wrapper/xenbase/CitrixPrepareForMigrationCommandWrapper.java

```diff
@@ -47,7 +47,7 @@ public Answer execute(final PrepareForMigrationCommand command, final CitrixReso
         String configDriveLabel = vm.getConfigDriveLabel();
 
         if (configDriveLabel == null)  {
-            configDriveLabel = "config";
+            configDriveLabel = "config-2";
```

Comment:
Why was the config drive label changed with a `-2` suffix?

Reply:
@rhtyd it's added to be OpenStack compatible. In this way it works out of the box in combination with cloud-init. [design doc config-drive](https://cwiki.apache.org/confluence/display/CLOUDSTACK/Using+ConfigDrive+for+Metadata%2C+Userdata+and+Password#UsingConfigDriveforMetadata,UserdataandPassword-OpenStack(cloud-init)compatibility(version2))